README.md

```diff
@@ -216,6 +222,46 @@ CustomScript uses the following algorithm to execute a script.
  1. write the decoded (and optionally decompressed) value to disk (/var/lib/waagent/custom-script/#/script.sh)
  1. execute the script using _/bin/sh -c /var/lib/waagent/custom-script/#/script.sh.
 
+### 1.5 managedIdentity
+
+CustomScript (version 2.1.2 onwards) supports [managed identity](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview) based RBAC for downloading file(s) from URLs provided in the "fileUris" setting. It allows CustomScript to  access Azure Storage private blobs/containers without the user having to pass secrets like SAS tokens or storage account keys.
```

Comment:
"private Azure Storage blobs or containers without having to pass in secrets like..."